packages/emails/templates/attendee-scheduled-email.ts

```diff
@@ -16,20 +16,18 @@ export default class AttendeeScheduledEmail extends BaseEmail {
   attendee: Person;
   showAttendees: boolean | undefined;
   t: TFunction;
-  attendees: Person[];
 
   constructor(calEvent: CalendarEvent, attendee: Person, showAttendees?: boolean | undefined) {
     super();
+    if (!showAttendees && calEvent.seatsPerTimeSlot) {
+      this.calEvent = cloneDeep(calEvent);
+      this.calEvent.attendees = [attendee];
+    } else {
+      this.calEvent = calEvent;
+    }
```

Comment:
@emrysal I implemented your feedback here.

Reply:
I was thinking the same about cloning inside the required function. This looks great now.

Reply:
It's the BaseEmail doing some kind of attendee filtering?

Reply:
If you're talking about `BaseScheduledEmail` we don't do any filtering on that level because the calendar event being passed into it is already filtered here.